components/assetcard/index.css

```diff
@@ -13,52 +13,84 @@ governing permissions and limitations under the License.
 /* outer container, unstyled */
 .spectrum-AssetCard {
   /* todo: this isn't quite the size from the XD file as 232px is not a size token, so we use 224px */
-  --spectrum-assetcard-asset-size: var(--spectrum-global-dimension-static-size-2800);
+  --spectrum-assetcard-asset-size: 224px;
+  --spectrum-assetcard-background-color: var(--spectrum-gray-200);
+  --spectrum-assetcard-asset-animation-duration: var(--spectrum-animation-duration-100);
+  --spectrum-assetcard-assetcontainer-border-size: 1px;
+  --spectrum-assetcard-header-margin-block-start: var(--spectrum-spacing-300);
```

Comment:
I wonder if this alias is slightly more accurate here than pointing directly to the spacing var? `var(--spectrum-text-to-visual-300)`